android/build-help.sh

```diff
@@ -72,7 +72,11 @@ displayInfo "" \
 cd $KEYMAN_ROOT/android/help
 
 MDLUA="$KEYMAN_ROOT/resources/build/html-link.lua"
-MD=`find -name "*.md"`
+if [[ "${OSTYPE}" == "darwin"* ]]; then
+  MD=`find . -name "*.md"`
+else
+  MD=`find -name "*.md"`
+fi
```

Comment:
```suggestion
MD=`find . -name "*.md"`
```

I think this should work everywhere